frontend/server/src/main/java/com/amazonaws/ml/mms/http/InferenceRequestHandler.java

```diff
@@ -157,7 +157,11 @@ private void predict(
         if (modelName == null) {
             modelName = input.getStringParameter("model_name");
             if (modelName == null) {
-                throw new BadRequestException("Parameter model_name is required.");
+                // Invocations will call the first loaded model if the modelName is not provided.
+                Map.Entry<String, Model> entry =
+                        ModelManager.getInstance().getModels().entrySet().iterator().next();
```

Comment:
This is probably wrong. When invocations is called, without a model name, we call the first model only if there is only one model loaded at start-up time. This might break things for some of our customers. The logic for that is just above, https://github.com/awslabs/mxnet-model-server/pull/852/files#diff-279625ee159eb456e707dc00653d323bR131.